src/crawlee/browsers/_browser_pool.py

```diff
@@ -100,30 +100,30 @@ def with_default_plugin(
         cls,
         *,
         browser_type: BrowserType | None = None,
-        browser_options: Mapping[str, Any] | None = None,
-        page_options: Mapping[str, Any] | None = None,
+        browser_launch_options: Mapping[str, Any] | None = None,
+        browser_new_context_options: Mapping[str, Any] | None = None,
         headless: bool | None = None,
         **kwargs: Any,
     ) -> BrowserPool:
         """Create a new instance with a single `PlaywrightBrowserPlugin` configured with the provided options.
 
         Args:
             browser_type: The type of browser to launch ('chromium', 'firefox', or 'webkit').
-            browser_options: Keyword arguments to pass to the browser launch method. These options are provided
+            browser_launch_options: Keyword arguments to pass to the browser launch method. These options are provided
                 directly to Playwright's `browser_type.launch` method. For more details, refer to the Playwright
                 documentation: https://playwright.dev/python/docs/api/class-browsertype#browser-type-launch.
-            page_options: Keyword arguments to pass to the page object is set at the playwright context level.
+            browser_new_context_options: Keyword arguments to pass to the page object is set at the playwright context.
                 These options are provided directly to Playwright's `browser.new_context` method. For more details,
                 refer to the Playwright documentation: https://playwright.dev/python/docs/api/class-browser#browser-new-context.
```

Comment:
Could we use the same as in the PW crawler?

```
            browser_launch_options: Keyword arguments to pass to the browser launch method. These options are provided
                directly to Playwright's `browser_type.launch` method. For more details, refer to the Playwright
                documentation: https://playwright.dev/python/docs/api/class-browsertype#browser-type-launch.
            browser_new_context_options: Keyword arguments to pass to the browser new context method. These options
                are provided directly to Playwright's `browser.new_context` method. For more details, refer to the
                Playwright documentation: https://playwright.dev/python/docs/api/class-browser#browser-new-context.
```

Reply:
Done